farmerbot/parser/validator.go

```diff
@@ -0,0 +1,44 @@
+package parser
+
+import (
+	"fmt"
+	"slices"
+
+	"github.com/threefoldtech/tfgrid-sdk-go/farmerbot/internal"
+)
+
+// ValidateInput validates that included, excluded and priority nodes are in the farm
+func ValidateInput(input *internal.Config, sub internal.Substrate) error {
```

Comment:
why using the address, why not just passing the value of the config?

Reply:
you are right, there is no need for it